README.md

```diff
@@ -30,15 +30,15 @@ As you continue to develop your applications, each view controller will become m
 
 ## See it in action with our example app
 
-Clone our repo, open `SwiftCurrent.xcworkspace`, target the `SwiftCurrentExample` scheme, and run to see our example app in action.
+Clone our repo, open `SwiftCurrent.xcworkspace`, target the `UIKitExample` scheme, and run to see our example app in action.
 
-The app is designed to give you an idea of what SwiftCurrent can do with minimal overhead in the UI.  The key areas of code you'll want to look at are: `SetupViewController.swift` and the view controllers referenced in the [Workflow](https://wwt.github.io/SwiftCurrent/Classes/Workflow.html).
+The app is designed to give you an idea of what SwiftCurrent can do with minimal overhead in the UI. [See the README](https://github.com/wwt/SwiftCurrent/blob/8e11ce58cc5410d09a40d43fefc0e722c87906b3/ExampleApps/UIKitExample/README.md) for the example project for suggestions of where to start looking in the project.
```

Comment:
This second sentence reads awkwardly. Not even sure I get what the message is that it's trying to say.

Should I be looking for a README in the example project for suggestions of where to start looking in the project?
Is it look at the README and the example project?
Or am I looking for references to the example project in the README?

Reply:
Yeah I'm not fully sure what I was trying to say either.  We now have README files in each example project that gives an overview of what the project is and interesting areas to start looking at.  For example, SwiftUI calls out the 3 main features, and then calls out ContentView as the entry point launching the workflows and being a tab item example.

So I'm trying to link to that README so people on Pages and people visiting Github know that there is a README.  Those who have opened the workspace and are looking at the Example project will see the README at the root level, inside the project.

Reply:
Unrelated to this issue, the 0.3 second change apparently was not really needed.  It was to make the flakey Model tests to pass more reliably, I believe.  We could change it back, though our next item to work is fixing those flakey tests.  Your call.

Reply:
How about:
```suggestion
The [example app has a README](https://github.com/wwt/SwiftCurrent/blob/8e11ce58cc5410d09a40d43fefc0e722c87906b3/ExampleApps/UIKitExample/README.md) that details interesting usages.
```